spk/salt-minion/Makefile

```diff
@@ -1,12 +1,13 @@
 SPK_NAME = salt-minion
-SPK_VERS = 3001.1
-SPK_REV = 1
+SPK_VERS = 3004
+SPK_REV  = 2
 SPK_ICON = src/salt-minion.png
 
-PIP = $(WORK_DIR)/../../../native/python3/work-native/install/usr/local/bin/pip
-BUILD_DEPENDS = cross/python3
-SPK_DEPENDS = "python3>=3.7"
-UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARMv5_ARCHS)
+SPK_DEPENDS = "python38"
+
+BUILD_DEPENDS  = cross/python38 cross/setuptools cross/pip
+BUILD_DEPENDS += cross/wheel
+BUILD_DEPENDS += cross/salt
```

Comment:
Again, migrating to python310 + removal of setuptools, pip and wheel.
Also salt can most probably be covered directly using requirements-cross.txt